### ISSUE TITLE
What is the "risk"?

### ISSUE THREAD

**Issue by timrourke** [DataDog/dd-trace-php#64]
The readme for this project states: `This project is experimental and under active development. Use it at your own risk.`.

I love DataDog. My employer uses it heavily as an APM tool, and to do application-level custom metrics for BI insights, etc. I would love to know what the specific risks are here. Since my employer is already a customer of your APM product, it would be nice to further instrument our application with this tracer.

However, I want to know what the specific risks _are_. Any advice you can offer?

**Comment by chuck** [comment]
I've updated the readme to be a bit less scary, but to answer your question:

- It's under active development and there are no API contracts yet; everything is subject to change at least up until beta, so any customization or integration may have to be updated to match future changes.

- Performance hasn't been audited yet. There are also not yet guarantees that the tracer won't introduce crashes, leak errors, or otherwise affect the app being traced.

Short version, we recommend testing the tracer in staging at the moment.
